+++
title = 'A mutational atlas for Parkin proteostasis'
date = 2024-10-24T12:45:40+02:00
draft = false
tags = ['molecular-biology', 'science']
+++
[Link to the article here](https://www.nature.com/articles/s41467-024-45829-4)

## The Parkin Protein

The Parkin protein plays a central role in marking proteins for degradation, and is therefore an essential part of the Protein Quality Control system in cells.
Specifically, the Parkin protein labels proteins in damaged mitochondria, and as such acts as a "mitochondria bouncer" regulating whether damaged mitochondria should be broken down and recycled, a process known as mitophagy.
When a Parkin protein encounters a protein queued for removal (a "target"), it binds a ubiquitin protein to the protein, kind of like giving it a "red card".
This ubiquitin "red card" alerts the proteasome, or the "protein trash compactor" that the protein should be degraded and recycled.
In this way, the Parkin protein is vital component for cells the keep proteostasis, or the balance of proteins within the cell.

Mutations in the Parkin protein are linked to mitochondrial dysfunction, as damaged mitochondria are not removed from cells when the Parkin protein doesn't function properly.
In turn, these mutations are associated with the neurodegenerative disorder known as Parkinsons' disease and cancer tumor formation.
It's therefore important to understand what effects different mutations have on the Parkin protein's structure, activity, and stability.
In the long run, mapping these mutations in the Parkin protein can help understand, predict, and treat diseases related to Parkin.


### Structure of Parkin
The Parkin protein is composed of a single chain of 465 amino acids, divided into 7 domains/sections of amino acids:

1. A Ubiquitin-like (UBL) domain, which keeps the Parkin protein from red-carding itself.
2. A disordered part containing the "activation element" (ACT)
3. A Really Interesting New Gene domain (RING0)
4. A RING-between-RING (RBR module), containing the "active site", composed of
   1. Another RING domain (RING1)
   2. An In-Between-RING domain (IBR)
   3. A repressor element (REP)
   4. Finally, the second RING domain (RING2) which contains the Cysteine (pos. 431) responsible for catalysis (the attachment of the "red-card").

### Protein Mutations
There are three types of naturally occuring protein mutations.
To understand them, it helps to understand the different levels of protein structure, which is wonderfully explained with this picture from [Introductory Biology](https://bio.libretexts.org/Courses/University_of_California_Davis/BIS_2A%3A_Introductory_Biology_(Britt)/01%3A_Readings/1.17%3A_Protein_Structure):
![Levels of Protein Structure](https://bio.libretexts.org/@api/deki/files/9689/quatranary.jpg?revision=1&size=bestfit&width=413&height=550)



## Experiments
The authors used a technique known as "deep mutational scanning" to investigate _every single substutution mutation_ in the Parkin protein.
They measured abundance in the Parkin protein, as a heuristic for stability and likelihood of degradation. 
Basically, if a given mutation has _low_ abundance, that means one of two things:
1. The mutated protein is for some reason difficult to _produce_ in the cell, so the protein breaks down. before it has formed.
2. The mutated protein is unstable and immediately red-carded and destroyed by the proteasome. 


## Results
### Effects of mutations on the Parkin protein abundance
![Figure 2 - A variant Effect Map of Parkin protein abundance](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-024-45829-4/MediaObjects/41467_2024_45829_Fig2_HTML.png?as=webp)



## The questions I'm left with after this article?
